client/packages/inventory/src/Stocktake/DetailView/modal/StocktakeLineEdit/StocktakeLineEditTables.tsx

```diff
@@ -160,6 +161,11 @@ const getInventoryAdjustmentReasonInputColumn = (
             isVaccine: rowData.item.isVaccine,
             isDispensary: store?.storeMode === StoreModeNodeType.Dispensary,
           })}
+          fallbackType={
+            isInventoryReduction
+              ? ReasonOptionNodeType.NegativeInventoryAdjustment
+              : ReasonOptionNodeType.PositiveInventoryAdjustment
+          }
```

Comment:
To explain more of my reservation about our data model I was yapping about earlier - when a user is doing reports or asking a a dashboard, they may try for filtering all reductions based on `NegativeInventoryAdjustment`, however `OpenVialWastage` and `ClosedVialWastage` are also essentially negative inventory adjustments...

Maybe it's ok though and you should just filter by the type of inventory adjustment, or `STOCK_OUT` on the lines...